test/functional/Jsr292/src/com/ibm/j9/jsr292/indyn/BootstrapMethods.java

```diff
@@ -247,7 +247,7 @@ static String dup(Integer i) {
 	}
 
 	static String dup(Object o) {
-		return "DoesNotUnderStand: " + o.getClass() + " message: double";
```

Comment:
copyrights on this file needs to be updated

Reply:
Fixed in 1a389d000076aab140d9052e49e770b7bc54c0fa.